src/odemis/odemisd/test/mdupdater_test.py

```
@@ -88,5 +88,89 @@ def fake_get_component(name):
+            }
+        )
+        cls.ebeam = next(c for c in cls.sem.children.value if c.role == "e-beam")
+
+        # Create a fake CCD that will be affected by the e-beam
+        img = model.DataArray(numpy.empty((512, 768), dtype=numpy.uint16))
+        cls.ccd = mock.FakeCCD(img)
+
+        cls.ebeam.affects.value = [cls.ccd.name, "se-detector"]
+
+        # Mock model.getComponent() so MetadataUpdater can resolve component names
+        all_comps = list(cls.sem.children.value) + [cls.sem, cls.ccd]
```

Ruff (0.15.15)

[warning] Consider `[*list(cls.sem.children.value), cls.sem, cls.ccd]` instead of concatenation

Replace with `[*list(cls.sem.children.value), cls.sem, cls.ccd]`

(RUF005)